kytos/core/atcp_server.py

```diff
@@ -89,7 +89,6 @@ class KytosServerProtocol(asyncio.Protocol):
     """
 
     known_ports = {
-        6633: 'openflow',
```

Comment:
Here, I would suggest you to keep the old one too. There is no need to remove from here, now.